From 667b86fb135c4ee16a3dc1978c170c42b43a0ae3 Mon Sep 17 00:00:00 2001
From: al <al@b3059339-0415-0410-9bf9-f77b7e298cf2>
Date: Tue, 24 Dec 2019 10:46:23 +0000
Subject: [PATCH 3/5] codecs.conf: Bump version

In SVN r38154 , ffmpeg12vdpau was removed and ffmpeg1vdpau and
ffmpeg2vdpau were added as replacement.

In previous commits AV1 codecs were added, and a lot more changes
were done in 8 years.

Bump codecs.conf, so users with a customized codecs.conf based on
an older version will fallback to the internal version and get a
message about having an outdated codecs.conf lying around.



git-svn-id: svn://git.mplayerhq.hu/mplayer/trunk@38157 b3059339-0415-0410-9bf9-f77b7e298cf2
---
 etc/codecs.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/codecs.conf b/etc/codecs.conf
index fa8f1fedb..f4ff4425b 100644
--- a/etc/codecs.conf
+++ b/etc/codecs.conf
@@ -3,7 +3,7 @@
 ;  Before editing this file, please read DOCS/tech/codecs.conf.txt !
 ;=============================================================================
 
-release 20111012
+release 20191203
 
 ;=============================================================================
 ;                   VIDEO CODECS
-- 
2.26.0

